packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -227,6 +229,49 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
     }
   };
 
+  lockBodyScroll() {
+    const openDialogs = (+this.context.target.body.getAttribute(Dialog.dialogsCounterAttribute) || 0) + 1;
+    this.context.target.body.setAttribute(Dialog.dialogsCounterAttribute, `${openDialogs}`);
+
+    // Avoid to block scroll in nested dialogs
+    if (openDialogs === 1) {
+      disableBodyScroll(this.context.target.scrollingElement);
+    }
+  }
+
+  unlockBodyScroll() {
+    const openDialogs = (+this.context.target.body.getAttribute(Dialog.dialogsCounterAttribute) || 0) - 1;
+    this.context.target.body.setAttribute(Dialog.dialogsCounterAttribute, `${openDialogs}`);
+
+    // Only enables scroll if all dialogs are closed
+    if (openDialogs === 0) {
+      enableBodyScroll(this.context.target.scrollingElement);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Document/scrollingElement

It's not supported in IE 😭

Reply:
Replaced by `body`